docs/suppressing-instrumentation.md

```diff
@@ -6,11 +6,99 @@ You can disable the agent using `-Dotel.javaagent.enabled=false`
 ## Suppressing specific agent instrumentation
 
 You can suppress agent instrumentation of specific libraries by using
-`-Dotel.instrumentation.[id].enabled=false`.
+`-Dotel.instrumentation.[name].enabled=false` where `name` is the instrumentation `name`
+(main or additional):
 
-where `id` is the instrumentation `id`:
-
-[TODO add table here with all instrumentation ids]
+| Instrumentation name |
+|---------------------------|
+| akka-actor|
```

Comment:
Single column table looks odd.

Maybe bullets?

Or maybe add "Library/Framework" column (matching https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks), e.g.


```suggestion
| Library/Framework | Instrumentation name |
|-------------|--------------|
| Akka HTTP | akka-actor |
```

Reply:
+1 for library name